package.yaml

```diff
@@ -22,6 +22,10 @@ dependencies:
   - transformers-base >= 0.4.5 && < 0.5
   - exceptions >= 0.10 && < 0.11
   - stm >= 2.5 && < 2.6
+  - th-abstraction >= 0.3 && < 0.5
+  - template-haskell >= 2.15 && < 2.19
+  - syb >= 0.7 && < 0.8
+  - containers >= 0.6 && < 0.7
```

Comment:
I completely forgot about the dependencies this would incur. I'll need to further mull over whether this would warrant a separate library for that reason -- I'll reach out to you tomorrow about it.